lib/logger/lib/logger.ex

```diff
@@ -971,6 +982,8 @@ defmodule Logger do
     Enum.any?(matching, fn filter ->
       Enum.all?(filter, fn
         {:level_lower_than, min_level} ->
+          min_level = Logger.Handler.elixir_level_to_erlang_level(min_level)
```

Comment:
Wait, compare_levels already calls `elixir_level_to_erlang_level`. So this line is probably not necessary at all?

Reply:
Yeah, I think that you are right. It shouldn't be a case there